### PR TITLE
Use python3-rohmu library instead of pghoard

### PR DESCRIPTION
**Pull request by raphlopez**
The [rohmu project](https://github.com/aiven/rohmu) was recently split from its parent project, [pghoard](https://github.com/aiven/pghoard). As myhoard relies on rohmu for performing backup tasks (object storage upload/download, compression, encryption), the package dependency must be updated. 

Resolves aiven/rohmu issue [#7](https://github.com/aiven/rohmu/issues/7)

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
Test failures seem related.

**Comment by rikonen** [comment]
Is there some reason why this adds the rohmu dependencies but does not remove references to pghoard?

**Comment by raphlopez** [comment]
There are a couple references to pghoard outside of rohmu (`pghoard.common.set_stream_nonblocking`, `pghoard.common.increase_pipe_capacity`). 
They both seem to be helper methods and are not highly coupled with anything else in pghoard. If it makes sense to copy them over into this project and remove pghoard dependency entirely, I can definitely do that.

**Comment by raphlopez** [comment]
Made [PR #48](https://github.com/aiven/rohmu/pull/48) in rohmu to duplicate pghoard utils. We'll need to wait for the merge and next rohmu release before this can be merged without any pghoard dependencies.

**Comment by alexole** [comment]
PR was merged, could you update this PR please? I need it to get into aiven-core for my ticket @raphlopez

File: myhoard/version.py
Comment (alanfranz-at-work) [inline myhoard/version.py:1]: This mustn't be committed. I don't know why this is not in .gitignore, though. I had already noticed and thought about this.

**Review by ghost — DISMISSED** [review]
LGTM beyond the version glitch.

File: myhoard/version.py
Comment (raphlopez) [reply myhoard/version.py]: Thanks for pointing that out. Removed the file and added it to the .gitignore.

File: requirements.txt
Comment (alexole) [inline requirements.txt:8]: It won't work with any version of `rohmu`, would be better to specify `rohmu >= 1.0.2`

File: myhoard.spec
Comment (alexole) [inline myhoard.spec:28]: Same here and above (see my comment about the version)

**Review by alexole — APPROVED** [review]
LGTM

**Pull request merged**
